docs/reference/treyresearch/armTemplates/portal-es-lite.json

```diff
@@ -322,27 +302,92 @@
 						"visible": "[equals(steps('esGoalState').esLogAnalytics, 'Yes')]"
                     },
                     {
-                        "name": "textBlock2",
+                        "name": "textBlock0",
                         "type": "Microsoft.Common.TextBlock",
-                        "visible": "[equals(steps('esGoalState').esLogAnalytics, 'Yes')]",
+                        "visible": true,
```

Comment:
This will not cater for the conditions specified earlier and leave a dangling text box if the option for Log Analytics is set to "No"

Reply:
Indeed. Will fix but using the same condition used by the other security options.